src/app/helpers/app-initializer/app-initializer.ts

```diff
@@ -49,6 +49,7 @@ export interface Settings {
     harvestFilenameGuide: string;
   };
   hideProjects: boolean;
+  websiteMaps: boolean;
```

Comment:
nit naming: suggest `globalMaps`

Reply:
Nit: "global" sounds geographical

if we change this name to `globalMaps`, it might be confusing because you can still have global geographical map with this environment setting turned off.

By a similar thought train, I also originally rejected `siteMaps` because the term "site" was overloaded with "region/site/point" terminology (which is why I originally landed on "websiteMaps").

I'm happy to keep `globalMaps`, just want to make a note that it might be confusing

Reply:
Fair. Happy with a different name. Even long or obtuse - hopefully this is just a temporary config.

`disableHomePageAndProjectListMaps`